### ISSUE TITLE
Request for Program 3779 (Advanced Comp Sci) to be added to the website.

### ISSUE THREAD

**Issue by nakkers808** [csesoc/Circles#1202]
My program (Bachelor of Advanced Computer Science) is not available on the planner. I hope it can be added shortly.

**Comment by lhvy** [comment]
Hi, this is already on the roadmap for us. We're unfortunately blocked on another PR but hoping to deploy a lot of changes soon, which should include 3779 support.
